packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts

```diff
@@ -779,6 +834,29 @@ describe('BasicTracerProvider', () => {
   });
 
   describe('.forceFlush()', () => {
+    it('should call forceFlush with the default processor', done => {
```

Comment:
note for reviewer: this is the test to check that with exporter/processor defined from env the `forceFlush` method is called.